docs/topics/spiders.rst

```diff
@@ -68,6 +68,28 @@ Spiders receive arguments in their constructors::
             self.start_urls = ['http://www.example.com/categories/%s' % category]
             # ...
 
+To pass more complex arguments like dicts and lists use ``arg-json`` instead.
+Each argument must be JSON-encoded. For example::
+
+    scrapy crawl myspider --arg-json start_urls=['http://www.example.com', 'http://dmoz.com']
```

Comment:
It won't fly:

* JSON quotes are double-quotes;
* quotes will be interpreted by shell, so the script won't receive any quotes;
* the arguments will be split my a whitespace.

The way to go is 

`scrapy crawl myspider --arg-json 'start_urls=["http://www.example.com", "http://dmoz.com"]'`

This error could be very comon; I think we should add a note about it to the docs.